### PR TITLE
Refactor: tokens supply sub

### PR DESCRIPTION
**Pull request by luixo**
We can significantly reduce amount of queries to the analytics database.
The only quirk here I see is timezones.
I'm not sure at what moment operate all the actors and we should update such statistics.

### CODE REVIEW

File: backend/src/cron/tasks.ts [hunk 1]
```diff
@@ -109,12 +110,17 @@ export const transactionCountHistoryCheck: RegularCheckFn = {
   },
 };
 
+export const tokensSupplyCheck: RegularCheckFn = {
+  description: "circulatingSupplyCheck",
+  fn: publishOnChange("tokensSupply", queryTokensSupply, () => {
+    const endOfDayTimestamp = endOfDay(new Date()).valueOf();
+    return endOfDayTimestamp - Date.now();
```
Comment (luixo) [inline backend/src/cron/tasks.ts:117]: @frol I guess "end of the day" is not the right heuristics for the update interval.
Currently database lags for 2 days so we don't stumble upon showing incomplete data for say today, but could we in the future?

Comment (frol) [reply backend/src/cron/tasks.ts:117]: Can we just update it hourly? Circulating supply is getting computed for the day before when it is a few minutes in after midnight UTC.I believe it is totally fine if we lag by 1 hour.

**Review by frol — APPROVED** [review]
Overall, this PR looks good to me (see my only comment response above)

File: backend/src/cron/tasks.ts (hunk 1 above)
Comment (luixo) [reply backend/src/cron/tasks.ts:117]: I guess we can

**Pull request merged**